Dockerfile

```diff
@@ -1,17 +1,20 @@
 ##########################################################
 # Core Mava image
-FROM registry.kao.instadeep.io/library/nvidia/cuda:11.4.2-cudnn8-runtime-ubuntu20.04 as mava-core
+FROM nvidia/cuda:11.4.2-cudnn8-runtime-ubuntu20.04 as mava-core
```

Comment:
I thought we were switching away from externally hosted images in favour of ones hosted within InstaDeep?

Reply:
The internally hosted image was down yesterday so I switched back the official nvidia cuda one. Previously, we had issues when making a lot of pulls to the official nvidia one, if those issues pop up again, we can switch back to an internally hosted one.